mls-rs-core/Cargo.toml

```diff
@@ -11,7 +11,9 @@ exclude = ["test_data"]
 
 
 [features]
-default = ["std", "rfc_compliant", "fast_serialize"]
+web = []
+node = []
+default = ["std", "rfc_compliant", "fast_serialize", "web"]
```

Comment:
I don't think this is needed because it is covered in mls-rs-crypto-webcrypto

Reply:
could you elaborate?

Reply:
are you referring to this: https://doc.rust-lang.org/cargo/reference/features.html#feature-unification ?

Reply:
cool thanks for pointing out, this helped me with finding a good solution :)